istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -325,6 +325,9 @@ func clusterConfigCmd(ctx cli.Context) *cobra.Command {
 		Example: `  # Retrieve summary about cluster configuration for a given pod from Envoy.
   istioctl proxy-config clusters <pod-name[.namespace]>
 
+  #  Retrieve summary about cluster configuration for a pod under a deployment
```

Comment:
Can you make the examples more consistent? I see some sentences include `.` while others do not, and some have `from envoy` and some don't.